tasks/src/text-to-image/data.ts

```diff
@@ -0,0 +1,34 @@
+import type { TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+    datasets: [
```

Comment:
The datasets do not necessarily have to be most used datasets on Hub, but rather we'd like a dataset that people can use to train general models on (e.g. COCO for object detection). That would be nice to have a dataset like that, you can get a general purpose dataset, you don't necessarily need to filter for it.

Reply:
Thank you for your feedback. I have pushed for the general purpose datasets. I also added the Midjourney dataset because it is currently the one of the famous models in text-to-image area. Especially for artwork.